wiki/entrypoint.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+# This file is a fork of GitHub Wiki Action plugin by Andrew Chen Wang
+# https://github.com/Andrew-Chen-Wang/github-wiki-action
+
+TEMP_CLONE_FOLDER="temp_wiki_$GITHUB_SHA"
+
+# This script assumes there is rsync and git installed in executing container.
```

Comment:
It's true for ubuntu20.04 [1] image which is used here.

[1] https://github.com/actions/virtual-environments/tree/main/images/linux

Reply:
should I add that information to that comment?

Reply:
It's not necessary for me.